src/sg/gov/moh/healthcert/1.0/interim-pdt-healthcert-schema.json

```diff
@@ -0,0 +1,437 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "title": "Interim PDT HealthCert Schema",
+  "required": ["id", "name", "validFrom", "fhirVersion", "fhirBundle", "issuers", "$template"],
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "Internal reference, usually serial number, of this document",
+      "examples": ["TEST001"]
+    },
+    "name": {
+      "type": "string",
+      "examples": ["HealthCert"]
+    },
+    "validFrom": {
+      "type": "string",
+      "format": "date-time",
+      "description": "Date from which the document is considered valid",
+      "examples": ["2021-04-27T03:18:51.972Z"]
+    },
+    "fhirVersion": {
+      "type": "string",
+      "examples": ["4.0.1"]
+    },
+    "fhirBundle": {
+      "properties": {
+        "resourceType": { "type": "string", "default": "Bundle" },
+        "type": { "type": "string", "default": "collection" },
+        "entry": {
+          "type": "array",
```

Comment:
makes sense to have `minItems`?

Reply:
Yes, we could set `minItems: 4`. However, would that also allow for things like 4 Patient objects? Instead of one of each (Patient, Specimen, Observation, Organisation).

Alternatively, would changing `anyOf` -> `allOf` help?

Reply:
Will not have it